src/fsharp/PatternMatchCompilation.fs

```diff
@@ -1434,3 +1422,5 @@ let rec CompilePattern  g denv amap tcVal infoReader exprm matchm warnOnUnused a
 
     | _ ->
         CompilePatternBasic g denv amap tcVal infoReader exprm matchm warnOnUnused true actionOnFailure (origInputVal, origInputValTypars, origInputExprOpt) clausesL inputTy resultTy
+type IA =
```

Comment:
I think this was an accident and should be removed.

Reply:
Well spotted!